cmd/spawn.go

```diff
@@ -50,6 +50,8 @@ results by reading the stdout of each agent benchmark.
 	flags.StringSliceVar(&spawn.AgentHosts, "agent-hosts", []string{"localhost"}, "Comma separated hostnames of agents.")
 	flags.StringVar(&spawn.SSHUser, "ssh-user", "", "SSH username for remote hosts. (defaults to current user)")
 	flags.StringVar(&spawn.SpawnFile, "spawn-file", "", "JSON file defining spawn benchmarks. (required)")
```

Comment:
we should change the doc string here to something like "Local JSON file containing benchmarks to spawn (only used if remote filename not set)."